stdlib/src/collections/string.mojo

```diff
@@ -686,6 +686,63 @@ fn isprintable(c: UInt8) -> Bool:
     return ord_space <= int(c) <= ord_tilde
 
 
+# ===----------------------------------------------------------------------=== #
+# String Representation
+# ===----------------------------------------------------------------------=== #
+
+
+struct StringRepr(Writable):
```

Comment:
**Question** Should we make this private or do you expect users to interact with this?  I'd default to private for now.